factory/__init__.py

```diff
@@ -8,7 +8,7 @@
 from command.core import Core
 from command.commands.token import TokenCommandConfig
 from datetime import timedelta
-from db.facade import DBFacade
+from db import DBFacade
 from db.dynamodb import DynamoDB
```

Comment:
and one for DynamoDB as well?

Reply:
I'm not sure we want to do that. The DynamoDB class is supposed to be an implementation detail, so I'm not convinced we want to reexport it at the `db` level.